classes/PodsAPI.php

```diff
@@ -6243,6 +6272,14 @@ public function load_pods( $params = null ) {
 			$params['id'] = $params['ids'];
 		}
 
+		$include_internal = false;
+
+		if ( isset( $params['include_internal'] ) ) {
+			$include_internal = (boolean) $params['include_internal'];
```

Comment:
:no_entry_sign: Short form type keywords must be used. Found: (boolean)

<sub>hash: f406a271ce3a82851443607cd87ad834</sub>